src/components/curator-actions/fetch-signatories.ts

```diff
@@ -16,11 +32,26 @@ type GraphQLResponse = {
 	};
 };
 
+type MultisigAccountResponse = {
```

Comment:
```suggestion
interface MultisigAccountResponse {
```
The rule of thumb I keep seeing is to only switch to types when interfaces cannot do something